### PR TITLE
fix: polish controller bar in collection and my nft page

### PR DESCRIPTION
**Pull request by lynnshaoyu**
https://uniswaplabs.atlassian.net/browse/WEB-1647?atlOrigin=eyJpIjoiZWIwMmU5ZTI3NzgxNDE5Mjg0MWIyYWViNDUzMmQ0ZGIiLCJwIjoiaiJ9

### CODE REVIEW

File: src/nft/components/collection/CollectionNfts.tsx [hunk 1]
```diff
@@ -301,39 +298,39 @@ export const CollectionNfts = ({ contractAddress, collectionStats, rarityVerifie
...
+              icon: <RarityIcon width="20" height="20" viewBox="2 2 24 24" color={vars.color.blue400} />,
               reverseIndex: 3,
             },
           ]
         : [
             {
               displayText: 'Low to High',
               onClick: () => setSortBy(SortBy.LowToHigh),
-              icon: <NonRarityIcon width="28" height="28" color={vars.color.blue400} />,
+              icon: <NonRarityIcon width="20" height="20" viewBox="2 2 22 22" color={vars.color.blue400} />,
```
Comment (vm) [inline src/nft/components/collection/CollectionNfts.tsx:327]: if these are re-used can you make a single instance at the top and then use the variable?

File: src/nft/components/collection/CollectionNfts.tsx [hunk 2]
```diff
@@ -462,16 +460,8 @@ export const CollectionNfts = ({ contractAddress, collectionStats, rarityVerifie
...
-                  <ThemedText.BodyPrimary
-                    fontWeight={600}
-                    color={sweepIsOpen && buyNow ? theme.white : theme.textPrimary}
-                    lineHeight="20px"
-                    marginTop="2px"
-                    marginBottom="2px"
-                  >
-                    Sweep
-                  </ThemedText.BodyPrimary>
+                  <SweepIcon viewBox="0 0 24 24" width="20px" height="20px" />
```
Comment (vm) [inline src/nft/components/collection/CollectionNfts.tsx:463]: i think the viewbox could be set based on width/height

File: src/nft/components/collection/FilterButton.tsx [hunk 3]
```diff
@@ -66,12 +67,15 @@ export const FilterButton = ({
           )}
 
           <FilterIcon
-            style={{ marginBottom: '-4px', paddingRight: `${!isFiltersExpanded || showFilterBadge ? '6px' : '0px'}` }}
+            style={{
+              marginBottom: '-4px',
+              paddingRight: `${!isFiltersExpanded || showFilterBadge ? '6px' : '0px'}`,
```
Comment (vm) [inline src/nft/components/collection/FilterButton.tsx:72]: did this change? or just prettier?

**Review by vm — APPROVED** [review]

File: src/nft/components/collection/CollectionNfts.tsx (hunk 2 above)
Comment (lynnshaoyu) [reply src/nft/components/collection/CollectionNfts.tsx:463]: nah they kinda vary across SVGs...there is not a uniform appropriate viewBox across icons. kind of annoying but we really have to tweak them individually to get it to look right.

File: src/nft/components/collection/CollectionNfts.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/nft/components/collection/CollectionNfts.tsx:327]: good idea done.

File: src/nft/components/collection/FilterButton.tsx (hunk 3 above)
Comment (lynnshaoyu) [reply src/nft/components/collection/FilterButton.tsx:73]: changed now, added some new stuff, but previously i think just prettier

**Pull request merged**